source/melpomene/src/sim_drivers/emb_display.rs

```diff
@@ -31,31 +31,15 @@ use embedded_graphics_simulator::{
 use maitake::sync::Mutex;
 use mnemos_alloc::containers::HeapArray;
 use mnemos_kernel::{
-    comms::{
-        kchannel::{KChannel, KConsumer},
-        oneshot::Reusable,
-    },
-    registry::{Envelope, KernelHandle, Message, RegisteredDriver, ReplyTo},
+    comms::kchannel::{KChannel, KConsumer},
+    drivers::emb_display::{EmbDisplayService, FrameChunk, FrameError, Request, Response},
+    registry::Message,
     Kernel,
 };
-use uuid::Uuid;
 
-//////////////////////////////////////////////////////////////////////////////
-// EmbDisplay - This is the "driver type"
-//////////////////////////////////////////////////////////////////////////////
-
-// Registered driver
-pub struct EmbDisplay;
-
-// impl EmbDisplay
-impl RegisteredDriver for EmbDisplay {
-    type Request = Request;
-    type Response = Response;
-    type Error = FrameError;
-    const UUID: Uuid = mnemos_kernel::registry::known_uuids::kernel::EMB_DISPLAY;
-}
+pub struct SimDisplay;
```

Comment:
maybe worth a comment like
```suggestion
/// Implements the [`EmbDisplayService`] driver using the `embedded-graphics`
/// simulator.
pub struct SimDisplay;
```